modules/genomic_browser/php/NDB_Menu_Filter_cnv_browser.class.inc

```diff
@@ -391,8 +391,9 @@ class NDB_Menu_Filter_CNV_Browser extends NDB_Menu_Filter
                 $query .= " AND $field = CONVERT( :v_$prepared_key, DECIMAL) ";
             } elseif ($field == "genomic_range") {
                 $temp_array = explode(':', $val);
-                $chr        = $temp_array[0];
-                $query     .= " AND genome_loc.Chromosome = '$chr'";
+                $chr        = str_replace('chr', '', $temp_array[0]);
+                $query     .= " AND (genome_loc.Chromosome = '$chr'";
+                $query     .= " OR genome_loc.Chromosome = 'chr$chr')";
```

Comment:
This appears to be taking user input and concatenating it into an SQL query, which is vulnerable to an SQL injection attack. It should at the very least be using the Database::quote function to properly escape the input if it can't use prepared statements.